src/components/EditableCard.tsx

```diff
@@ -81,8 +80,8 @@ class EditableCard extends Component<IProps> {
   }
 
   private get editButton () {
-    const { cardConfig, isLoading, isGuarded } = this.propsWithDefaults
-      , classNameSuffix = cardConfig.classNameSuffix || kebabCase(cardConfig.title);
+    const { isLoading, title, isGuarded } = this.propsWithDefaults
+      , classNameSuffix = this.propsWithDefaults.classNameSuffix || kebabCase(title);
```

Comment:
this could be pulled out into a getter since it's used identically twice

Reply:
Agree, but this PR was pretty complex and I didn't want to push it beyond the PR title